src/main/resources/config.yml

```diff
@@ -1,4 +1,8 @@
 potion-duration: 10 # in minutes
 region-name: km_safe_from_radiation
+# Set this to false when you want to create safe region by yourself
+use-radius: true
+#Radius from spawnpoint, where safe region should be
```

Comment:
Formatowanie jest niezgodne z linijką 3. Poza tym wydaje mi się, że opisy powinny być po polsku.

Reply:
@TheMolkaPL w linii pierwszej komentarz `# in minutes` jest po angielsku